### DISCUSSION TITLE
version `GLIBC_2.38' not found

### DISCUSSION THREAD

**Discussion by overtube** [rbenv/rbenv#1524] [Q&A]
`rbenv exec gem install sidekiq --verbose`
```
/home/ui/.rbenv/versions/3.2.2/bin/ruby: /usr/lib/libm.so.6: version `GLIBC_2.38' not found (required by /home/ui/.rbenv/versions/3.2.2/lib/libruby.so.3.2)
/home/ui/.rbenv/versions/3.2.2/bin/ruby: /usr/lib/libc.so.6: version `GLIBC_2.38' not found (required by /home/ui/.rbenv/versions/3.2.2/lib/libruby.so.3.2)
```
**OS:** `x86_64`
`pacman -Qi rbenv`
Name            : rbenv
Version         : 1.2.0-1
Description     : Simple Ruby version manager
Architecture    : x86_64
URL             : https://github.com/rbenv/rbenv
Licenses        : MIT
Groups          : None
Provides        : None
Depends On      : None
Optional Deps   : ruby-build [installed]
Required By     : None
Optional For    : None
Conflicts With  : rbenv-git
Replaces        : None
Installed Size  : 51.98 KiB
Packager        : Unknown Packager
Build Date      : Thu 07 Sep 2023 11:11:16 PM EEST
Install Date    : Thu 07 Sep 2023 11:11:34 PM EEST
Install Reason  : Explicitly installed
Install Script  : Yes
Validated By    : None```

**Comment by overtube** [comment]
also I reinstall on using git version, result similar, and symlnk exist

`pacman -Q rbenv-git`
`rbenv-git 1.2.0.r72.g5d78e47-1`

`pacman -Q ruby-build-git`
`ruby-build-git v20230904-1`

**Comment by hsbt** [comment]
It seems to conflict with your glibc version and ruby installation. Can you try with `rbenv install` again?

**Reply by overtube** [reply]
Y, and now all work nice, thx
[build-log.txt](https://github.com/rbenv/rbenv/files/12556857/build-log.txt)


[bash_profile.txt](https://github.com/rbenv/rbenv/files/12556850/bash_profile.txt)
[bashrc.txt](https://github.com/rbenv/rbenv/files/12556851/bashrc.txt)

[gem-sidekiq-build.log.txt](https://github.com/rbenv/rbenv/files/12557038/gem-sidekiq-build.log.txt)
